src/jumio-kyc/kyc.service.ts

```diff
@@ -274,13 +274,13 @@ export class KycService {
         : this.redemptionService.calculateStandaloneWatchlistStatus(status);
 
     // Has our user's KYC status changed
```

Comment:
We now need to update even if user status goes from `TRY_AGAIN` -> `TRY_AGAIN` since this could be caused by age issue.

Reply:
If the user fails because their age < 18, the return status will be `TRY_AGAIN`. If they were already in `TRY_AGAIN` status, without this change we wouldn't persist their age.

I believe this guard was to prevent reprocessing of callbacks.